src/common/translations/en.json

```diff
@@ -305,6 +305,10 @@
       "title": "Too Many Words",
       "body": "The recovery phrase has to be 12 words."
     },
+    "amountExceeds": {
+      "title": "Amount Exceeds",
```

Comment:
title should be "Not enough balance"
"body": "The amount entered exceeds the wallet's balance"

Please update other languages too.